CHANGELOG.md

```diff
@@ -1,3 +1,238 @@
+### v0.6.0 (2023-08-09)
+
+* **BREAKING** - Dropped support for Node 14.
+
+* Updated instrumentation to no longer record spans for middleware execution.

```

Comment:
I forgot to include this in my PR but I would call out that middleware instrumentation only runs on Next.js 12.2.0-13.4.12